src/liftwing_api/models/revertrisk.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+from liftwing_api.models.liftwing_model import LiftwingModel
+from typing import Dict
+
+class RevertRiskAPIModel(LiftwingModel):
+    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict"):
+        super().__init__(base_url)
+
+    def request(self, payload: Dict[str, int], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, int]:
+        """
+        This function makes a POST request to https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict
+        using the language parameter and returns a JSON
+        language is for the different wiki languages, rev_id is the specific revisions
+        """
+        language = payload.get("lang")
+        if language is None:
+            raise ValueError("'lang' parameter is required in the payload.")
+        rev_id = payload.get("rev_id")
+        if rev_id is None:
+            raise ValueError("rev id is none, add revision id to continue")
```

Comment:
nit: the underscore `_` is missing from the message. It is better for us to be specific in the error message, that way the user will know what to do exactly, without having to look at documentation etc.
```suggestion
            raise ValueError("rev_id is none, add revision id to continue")
```

Reply:
updated